lib/database/achievement.php

```diff
@@ -248,6 +249,11 @@ function UploadNewAchievement(
         return false;
     }
 
+    if (!AchievementPoints::isValid((int) $points)) {
+        $errorOut = "Invalid points value (" . $points . ").";
+        return false;
+    }
```

Comment:
@Jamiras haven't tested yet but i hope this play nicely with the toolkit?
It validates the points value globally with that addition.

Reply:
See https://github.com/RetroAchievements/RAWeb/pull/1054/files#diff-2553fd75e71c70fd2914740c5ea8c486a7c21f93496f8b785c94102c36537ea6R7

Reply:
Just tested it locally in RALib and it seems to give this error when trying to upload an invalid point which seems pretty nice to me
![image](https://user-images.githubusercontent.com/4047771/183312405-b13b022b-7962-47e2-b2de-172468f51d57.png)

Reply:
Awesome, thanks for testing!